src/pyodide.js

```diff
@@ -2,6 +2,13 @@
  * The main bootstrap script for loading pyodide.
  */
 
+// Regexp for validating package name and URI
+var package_name_regexp = '[a-z0-9_\-]+'
+var package_uri_regexp = new RegExp(
+     '^https?://.*?(' + package_name_regexp + ').js$', 'i');
```

Comment:
Initially I though that e.g. `file://` would also work here, but it doesn't seem to be valid. Not sure what should be the approach to load `.js` files from local system.

Reply:
Let's add this to the docs.